torchao/quantization/quant_api.py

```diff
@@ -855,6 +907,19 @@ def _int8_symm_per_token_reduced_range_quant_cutlass(
     )
 
 
+def _int4_symm_per_token_quant_cutlass(x: torch.Tensor) -> torch.Tensor:
+    return to_affine_quantized_intx(
+        x,
+        mapping_type=MappingType.SYMMETRIC,
+        block_size=_get_per_token_block_size(x),
+        target_dtype=torch.int8,
+        quant_min=-8,
+        quant_max=7,
+        eps=1e-5,
+        _layout=CutlassInt4PackedLayout(),
```

Comment:
you can set zero_point_domain to ZeroPointDomain.NONE to skip generating zero_point Tensor btw, same for weight

Reply:
oh sorry, it's added here: https://github.com/pytorch/ao/pull/1556 but not landed yet. this will be supported after it is landed

Reply:
Done